marklogic-data-hub-central/ui/src/components/sidebar/sidebar.test.tsx

```diff
@@ -4,47 +4,70 @@ import Sidebar from './sidebar';
 import { entityFromJSON, entityParser } from '../../util/data-conversion';
 import modelResponse from '../../assets/mock-data/model-response';
 import searchPayloadFacets from '../../assets/mock-data/search-payload-facets';
+import { render, fireEvent } from '@testing-library/react';
+
 
 describe("Sidebar component", () => {
-    let wrapper;
-    const parsedModelData = entityFromJSON(modelResponse);
-    const entityDefArray = entityParser(parsedModelData);
-
-    describe('All Entities Dropdown Option', () => {
-      beforeEach(() => {
-        wrapper = shallow(
-          <Sidebar
-            entityDefArray={entityDefArray}
-            facets={searchPayloadFacets}
-            selectedEntities={[]}
-            facetRender = {jest.fn()}
-            checkFacetRender = {jest.fn()}
-          />
-        )
-      });
+  let wrapper;
+  const parsedModelData = entityFromJSON(modelResponse);
+  const entityDefArray = entityParser(parsedModelData);
 
-      it('should render only Hub Properties Panel', () => {
-        expect(wrapper.exists('#hub-properties')).toBe(true);
-        expect(wrapper.exists('#entity-properties')).toBe(false);
-      });
+  describe('All Entities Dropdown Option', () => {
+    beforeEach(() => {
+      wrapper = shallow(
+        <Sidebar
+          entityDefArray={entityDefArray}
+          facets={searchPayloadFacets}
+          selectedEntities={[]}
+          facetRender={jest.fn()}
+          checkFacetRender={jest.fn()}
+        />
+      )
     });
 
-    describe('An Entity is selected', () => {
-      beforeEach(() => {
-        wrapper = shallow(
-          <Sidebar
-            entityDefArray={entityDefArray}
-            facets={searchPayloadFacets}
-            selectedEntities={['Customer']}
-            facetRender = {jest.fn()}
-            checkFacetRender = {jest.fn()}
-          />
-        )
-      });
+    it('should render only Hub Properties Panel', () => {
+      expect(wrapper.exists('#hub-properties')).toBe(true);
+      expect(wrapper.exists('#entity-properties')).toBe(false);
+    });
+  });
```

Comment:
We can delete this suite as well lines 15-32. and the suite below with enzyme test.
The 2 tests here is when an entity is selected and when All Entities is selected and I think we can have 2 tests in 1 describe, than having 2 describes within a describe.
@timur-isangulov can you pls clean this up a little and add an RTL test for All entities similar to your RTL test for Customer.

Reply:
Done. Thanks.